Building error when run 'esy bootstrap' on windows 10

When Building front-end and run 'esy bootstrap', an error:
```
creating pathdef.c
    x86_64-w64-mingw32-gcc -c -I. -Iproto -DWIN32 -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -DHAVE_PATHDEF -DFEAT_NORMAL -DHAVE_STDINT_H -DMS_WIN64 -DHAVE_GETTEXT -DHAVE_LOCALE_H -DDYNAMIC_GETTEXT -DFEAT_MBYTE_IME -DDYNAMIC_IME -DDYNAMIC_ICONV -pipe -march=x86-64 -Wall -Wextra -Wno-strict-overflow -Wno-pointer-sign -Wno-unused-parameter -Werror -O3 -fomit-frame-pointer -freg-struct-return pathdef.c -o objx86-64/pathdef.o
    pathdef.c:6:32: error: missing terminating " character [-Werror]
     char_u *all_lflags = (char_u *)"x86_64-w64-mingw32-gcc -I. -Iproto -DWIN32 -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -DHAVE_PATHDEF -DFEAT_NORMAL -DHAVE_STDINT_H -DMS_WIN64 -DHAVE_GETTEXT -DHAVE_LOCALE_H -DDYNAMIC_GETTEXT -DFEAT_MBYTE_IME -DDYNAMIC_IME -DDYNAMIC_ICONV -pipe -march=x86-64 -Wall -Wextra -Wno-strict-overflow -Wno-pointer-sign -Wno-unused-parameter -Werror -O3 -fomit-frame-pointer -freg-struct-return -s -municode -o vim.exe -lkernel32 -luser32 -lgdi32 -ladvapi32 -lcomdlg32 -lcomctl32 -lnetapi32 -lversion -lole32 -luuid      C:\Program Files (x86)\VMware\CobraWinLDTP\ldtp\";
                                    ^
    pathdef.c:6:32: error: missing terminating " character
     char_u *all_lflags = (char_u *)"x86_64-w64-mingw32-gcc -I. -Iproto -DWIN32 -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -DHAVE_PATHDEF -DFEAT_NORMAL -DHAVE_STDINT_H -DMS_WIN64 -DHAVE_GETTEXT -DHAVE_LOCALE_H -DDYNAMIC_GETTEXT -DFEAT_MBYTE_IME -DDYNAMIC_IME -DDYNAMIC_ICONV -pipe -march=x86-64 -Wall -Wextra -Wno-strict-overflow -Wno-pointer-sign -Wno-unused-parameter -Werror -O3 -fomit-frame-pointer -freg-struct-return -s -municode -o vim.exe -lkernel32 -luser32 -lgdi32 -ladvapi32 -lcomdlg32 -lcomctl32 -lnetapi32 -lversion -lole32 -luuid      C:\Program Files (x86)\VMware\CobraWinLDTP\ldtp\";
                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    pathdef.c:7:1: error: expected expression before 'char_u'
     char_u *compiled_user = (char_u *)"Lee";
     ^~~~~~
    cc1: all warnings being treated as errors
    make: *** [Make_cyg_ming.mak:897: objx86-64/pathdef.o] Error 1
    error: command failed: "bash" "-c" "build/build-windows.sh" (exited with 2)
    esy-build-package: exiting with errors above...

  building libvim@8.10869.21
esy: exiting due to errors above
```

--- a/bench.esy.lock/index.json
+++ b/bench.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "2bc3274742f69aa979df33a7eae46083",
+  "checksum": "b0d032c25674db74e67ff4df0ee54b46",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "xmldom@0.1.27@d41d8cd9": {
@@ -264,18 +264,18 @@
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
     },
-    "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9": {
-      "id": "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+    "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9": {
+      "id": "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
       "name": "reason-libvim",
-      "version": "github:onivim/reason-libvim#5c86fee",
+      "version": "github:onivim/reason-libvim#8fc399d",
       "source": {
         "type": "install",
-        "source": [ "github:onivim/reason-libvim#5c86fee" ]
+        "source": [ "github:onivim/reason-libvim#8fc399d" ]
       },
       "overrides": [],
       "dependencies": [
         "refmterr@3.2.2@d41d8cd9", "ocaml@4.7.1004@d41d8cd9",
-        "libvim@8.10869.21@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
+        "libvim@8.10869.22@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
         "@esy-ocaml/reason@github:facebook/reason#474900e@d41d8cd9"
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
@@ -508,14 +508,14 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "libvim@8.10869.21@d41d8cd9": {
-      "id": "libvim@8.10869.21@d41d8cd9",
+    "libvim@8.10869.22@d41d8cd9": {
+      "id": "libvim@8.10869.22@d41d8cd9",
       "name": "libvim",
-      "version": "8.10869.21",
+      "version": "8.10869.22",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.21.tgz#sha1:9e652a01043dc22fd8bd7ef05476d8ffc36a6ca4"
+          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.22.tgz#sha1:ba7408c5a7ff2c6a99acce04a6083902389166bf"
         ]
       },
       "overrides": [],
@@ -906,7 +906,7 @@
         "rench@github:bryphe/rench#4860ef4@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#63d4056@d41d8cd9",
         "reason-tree-sitter@github:onivim/reason-tree-sitter#4947ead@d41d8cd9",
-        "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+        "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
         "reason-jsonrpc@1.1.0@d41d8cd9", "reason-glfw@3.2.1027@d41d8cd9",
         "ocaml@4.7.1004@d41d8cd9", "isolinear@2.0.0@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9", "axios@0.19.0@d41d8cd9",

--- a/esy.lock/index.json
+++ b/esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "2bc3274742f69aa979df33a7eae46083",
+  "checksum": "b0d032c25674db74e67ff4df0ee54b46",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "xmldom@0.1.27@d41d8cd9": {
@@ -264,18 +264,18 @@
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
     },
-    "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9": {
-      "id": "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+    "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9": {
+      "id": "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
       "name": "reason-libvim",
-      "version": "github:onivim/reason-libvim#5c86fee",
+      "version": "github:onivim/reason-libvim#8fc399d",
       "source": {
         "type": "install",
-        "source": [ "github:onivim/reason-libvim#5c86fee" ]
+        "source": [ "github:onivim/reason-libvim#8fc399d" ]
       },
       "overrides": [],
       "dependencies": [
         "refmterr@3.2.2@d41d8cd9", "ocaml@4.7.1004@d41d8cd9",
-        "libvim@8.10869.21@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
+        "libvim@8.10869.22@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
         "@esy-ocaml/reason@github:facebook/reason#474900e@d41d8cd9"
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
@@ -508,14 +508,14 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "libvim@8.10869.21@d41d8cd9": {
-      "id": "libvim@8.10869.21@d41d8cd9",
+    "libvim@8.10869.22@d41d8cd9": {
+      "id": "libvim@8.10869.22@d41d8cd9",
       "name": "libvim",
-      "version": "8.10869.21",
+      "version": "8.10869.22",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.21.tgz#sha1:9e652a01043dc22fd8bd7ef05476d8ffc36a6ca4"
+          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.22.tgz#sha1:ba7408c5a7ff2c6a99acce04a6083902389166bf"
         ]
       },
       "overrides": [],
@@ -905,7 +905,7 @@
         "rench@github:bryphe/rench#4860ef4@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#63d4056@d41d8cd9",
         "reason-tree-sitter@github:onivim/reason-tree-sitter#4947ead@d41d8cd9",
-        "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+        "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
         "reason-jsonrpc@1.1.0@d41d8cd9", "reason-glfw@3.2.1027@d41d8cd9",
         "ocaml@4.7.1004@d41d8cd9", "isolinear@2.0.0@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9", "axios@0.19.0@d41d8cd9",

--- a/integrationtest.esy.lock/index.json
+++ b/integrationtest.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "2bc3274742f69aa979df33a7eae46083",
+  "checksum": "b0d032c25674db74e67ff4df0ee54b46",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "xmldom@0.1.27@d41d8cd9": {
@@ -264,18 +264,18 @@
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
     },
-    "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9": {
-      "id": "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+    "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9": {
+      "id": "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
       "name": "reason-libvim",
-      "version": "github:onivim/reason-libvim#5c86fee",
+      "version": "github:onivim/reason-libvim#8fc399d",
       "source": {
         "type": "install",
-        "source": [ "github:onivim/reason-libvim#5c86fee" ]
+        "source": [ "github:onivim/reason-libvim#8fc399d" ]
       },
       "overrides": [],
       "dependencies": [
         "refmterr@3.2.2@d41d8cd9", "ocaml@4.7.1004@d41d8cd9",
-        "libvim@8.10869.21@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
+        "libvim@8.10869.22@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
         "@esy-ocaml/reason@github:facebook/reason#474900e@d41d8cd9"
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
@@ -508,14 +508,14 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "libvim@8.10869.21@d41d8cd9": {
-      "id": "libvim@8.10869.21@d41d8cd9",
+    "libvim@8.10869.22@d41d8cd9": {
+      "id": "libvim@8.10869.22@d41d8cd9",
       "name": "libvim",
-      "version": "8.10869.21",
+      "version": "8.10869.22",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.21.tgz#sha1:9e652a01043dc22fd8bd7ef05476d8ffc36a6ca4"
+          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.22.tgz#sha1:ba7408c5a7ff2c6a99acce04a6083902389166bf"
         ]
       },
       "overrides": [],
@@ -905,7 +905,7 @@
         "rench@github:bryphe/rench#4860ef4@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#63d4056@d41d8cd9",
         "reason-tree-sitter@github:onivim/reason-tree-sitter#4947ead@d41d8cd9",
-        "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+        "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
         "reason-jsonrpc@1.1.0@d41d8cd9", "reason-glfw@3.2.1027@d41d8cd9",
         "ocaml@4.7.1004@d41d8cd9", "isolinear@2.0.0@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9", "axios@0.19.0@d41d8cd9",

--- a/package.json
+++ b/package.json
@@ -216,7 +216,7 @@
     "@opam/ocaml-migrate-parsetree": "1.3.1",
     "@opam/ppx_tools_versioned": "5.2.2",
     "revery": "github:revery-ui/revery#af3599c",
-    "reason-libvim": "github:onivim/reason-libvim#5c86fee",
+    "reason-libvim": "github:onivim/reason-libvim#8fc399d",
     "reason-fontkit": "2.5.0",
     "@opam/camomile": "1.0.1",
     "@opam/ocamlfind": "1.8.0",

--- a/test.esy.lock/index.json
+++ b/test.esy.lock/index.json
@@ -1,5 +1,5 @@
 {
-  "checksum": "34f4f6dee1824a5fddb537639180715c",
+  "checksum": "d5a24d08b5c947ca125287fc06d1abfd",
   "root": "Oni2@link-dev:./package.json",
   "node": {
     "xmldom@0.1.27@d41d8cd9": {
@@ -264,18 +264,18 @@
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
     },
-    "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9": {
-      "id": "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+    "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9": {
+      "id": "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
       "name": "reason-libvim",
-      "version": "github:onivim/reason-libvim#5c86fee",
+      "version": "github:onivim/reason-libvim#8fc399d",
       "source": {
         "type": "install",
-        "source": [ "github:onivim/reason-libvim#5c86fee" ]
+        "source": [ "github:onivim/reason-libvim#8fc399d" ]
       },
       "overrides": [],
       "dependencies": [
         "refmterr@3.2.2@d41d8cd9", "ocaml@4.7.1004@d41d8cd9",
-        "libvim@8.10869.21@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
+        "libvim@8.10869.22@d41d8cd9", "@opam/dune@opam:1.7.3@e23fe0a7",
         "@esy-ocaml/reason@github:facebook/reason#474900e@d41d8cd9"
       ],
       "devDependencies": [ "ocaml@4.7.1004@d41d8cd9" ]
@@ -508,14 +508,14 @@
       "dependencies": [],
       "devDependencies": []
     },
-    "libvim@8.10869.21@d41d8cd9": {
-      "id": "libvim@8.10869.21@d41d8cd9",
+    "libvim@8.10869.22@d41d8cd9": {
+      "id": "libvim@8.10869.22@d41d8cd9",
       "name": "libvim",
-      "version": "8.10869.21",
+      "version": "8.10869.22",
       "source": {
         "type": "install",
         "source": [
-          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.21.tgz#sha1:9e652a01043dc22fd8bd7ef05476d8ffc36a6ca4"
+          "archive:https://registry.npmjs.org/libvim/-/libvim-8.10869.22.tgz#sha1:ba7408c5a7ff2c6a99acce04a6083902389166bf"
         ]
       },
       "overrides": [],
@@ -905,7 +905,7 @@
         "rench@github:bryphe/rench#4860ef4@d41d8cd9",
         "reasonFuzz@github:CrossR/reasonFuzz#63d4056@d41d8cd9",
         "reason-tree-sitter@github:onivim/reason-tree-sitter#4947ead@d41d8cd9",
-        "reason-libvim@github:onivim/reason-libvim#5c86fee@d41d8cd9",
+        "reason-libvim@github:onivim/reason-libvim#8fc399d@d41d8cd9",
         "reason-jsonrpc@1.1.0@d41d8cd9", "reason-glfw@3.2.1027@d41d8cd9",
         "ocaml@4.7.1004@d41d8cd9", "isolinear@2.0.0@d41d8cd9",
         "esy-macdylibbundler@0.4.5@d41d8cd9", "axios@0.19.0@d41d8cd9",

